src/data-table/index.js

```diff
@@ -280,7 +288,10 @@ class DataTable extends Component {
             </Table.Header>
             <Table.Body>
               {paginatedData.map(row => (
-                <Table.Row key={row._id}>
+                <Table.Row
+                  key={row._id}
+                  active={isRowActive && isRowActive(row)}
+                  onClick={onRowClick && this.createRowClickListener(row)}>
```

Comment:
it's more thorough to check if `onRowClick` is a function (same goes for `isRowActive`)

```suggestion
                  onClick={typeof onRowClick === 'function' && this.createRowClickListener(row)}>
```